engine/engine_test.go

```diff
@@ -1500,6 +1500,16 @@ func TestQueriesAgainstOldEngine(t *testing.T) {
 			end:   time.Unix(3000, 0),
 			step:  2 * time.Second,
 		},
+		{
+			name: "absent",
+			load: `load 30s
+				absent{pod="nginx-1", series="1"} 0
+				absent{pod="nginx-2", series="1"} 0`,
+			query: "absent(absent)",
```

Comment:
Can you add another test case where the metric doesn't exist? Also let's rename the metric name to not be absent